public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -188,15 +188,18 @@ export const getDefaultQueries = (): AlertQuery[] => {
         hide: false,
       },
     },
-    getDefaultExpression('B'),
+    ...getDefaultExpressions('B', 'C'),
   ];
 };
 
-const getDefaultExpression = (refId: string): AlertQuery => {
-  const model: ExpressionQuery = {
-    refId,
+const getDefaultExpressions = (...refIds: [string, string]): AlertQuery[] => {
```

Comment:
💬 I've changed the default to a chain of "reduce + math" expression for new Grafana managed alerts.

Reply:
I wonder if we should also change the default expression that gets added when you click `+ New expression`? Currently, we add the classic condition expression

Reply:
Good question, I think it would make sense – perhaps we should use the  `reduce` expression as the default, or the `threshold`? What do you think?